jsapp/js/projects/projectsTable/projectsTableRow.module.scss

```diff
@@ -119,19 +108,82 @@ $projects-table-hover-bg: colors.$kobo-gray-98;
     &[data-field='submissions'] {
       text-overflow: clip;
     }
+
+    // To make a more forgiving checkbox interaction, make this cell
+    // much wider than it otherwise would be.
+    &[data-field='checkbox'] {
+      position: relative;
+      // create hanging pseudo-elements to expand touch targets
+      &::after,
+      &::before {
+        display: inline-block;
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        z-index: 2;
+      }
+      // right: cover whitespace up to the project name link
+      &::after {
+        right: 0 - sizes.$x18;
+        width: sizes.$x18;
+      }
+      // left: cover whitespace left of project table
+      &::before {
+        left: 0 - sizes.$x32;
+        width: sizes.$x32;
+      }
+      // uncrop element
+      overflow: visible;
+      contain: inline-size style;
+
+      :global input.checkbox__input { // general appearance tweaks
+        &::after {border-radius: 1px;} // better-looking inner corners
+        &:focus-visible { // add keyboard focus styles
+          outline: 3px solid colors.$kobo-alt-blue;
```

Comment:
Could we add this in `/jsapp/js/components/common/checkbox-and-radio.scss` instead? I think it would be beneficial to all checkboxes :)

Reply:
Sure. After your radio/checkbox update, that's `checkbox.scss` and `radio.scss`.